src/manipulation.js

```diff
@@ -283,10 +283,14 @@ jQuery.extend({
 							}
 						}
 					}
-					delete elem[ dataPriv.expando ];
+					// Support: Chrome <=35 - 45
+					// Assign undefined instead of using delete, see Data#remove
```

Comment:
Do we want the full comment copied here and below? Seemed like a lot to put in 3 places...